roles/admin-users/tasks/main.yml

```diff
@@ -44,4 +44,10 @@
     exclusive: yes
   with_items: "{{ local_admin_users }}"
   become: true
+- name: Set selinux in permissive mode
```

Comment:
Should this not be handled at the beginning of this main.yml as opposed to at the end? Home dirs in /admin are created in the middle, keys are stored in ~/.ssh/authorized_keys and the sudoers config is modified too all of which may be denied by selinux...